package.json

```diff
@@ -3,6 +3,9 @@
   "version": "0.8.6",
   "license": "ISC",
   "main": "resolvewithplus.js",
+  "exports": {
+    "import": "./resolvewithplus.js"
+  },
```

Comment:
off-topic: given it's an esm-only package, it's recommended to add a `pkg.engines.node`, so package managers can output a waning when a user trying to install it on a unsupported node.js.

Reply:
the documentation gives node 12.16 as the version where support is added for exports.import, so adding this
```json
{
  "engines" : { 
    "node" : ">=12.16.0"
  }
}
```

Reply:
node v12/v13 has been EOL. maybe consider dropping node <14 at some point?

Reply:
ok the change can be added at https://github.com/iambumblehead/resolvewithplus/pull/26

Reply:
please note dropping node <14 will be a breaking change.